apps/cowswap-frontend/src/modules/bridge/pure/BridgeDetailsContainer/index.tsx

```diff
@@ -96,23 +90,9 @@ interface TitleActionsComponentProps {
   isExpanded?: boolean
 }
 
-function TitleActionsComponent({
-  explorerUrl,
-  isCollapsible = false,
-  isExpanded = false,
-}: TitleActionsComponentProps): ReactNode {
+function TitleActionsComponent({ isCollapsible = false, isExpanded = false }: TitleActionsComponentProps): ReactNode {
   return (
     <>
-      {explorerUrl && (
```

Comment:
We remove the 'view details' from the Bridge stop title, given it's the same link we now show in the 'Bridge transaction' line item.